lib/utils/docker.ts

```diff
@@ -193,19 +193,7 @@ export async function createClient(
 	opts: dockerode.DockerOptions,
 ): Promise<dockerode> {
 	const Docker = await import('dockerode');
-	const docker = new Docker(opts);
-	const { modem } = docker;
-	// Workaround for a docker-modem 2.0.x bug where it sets a default
```

Comment:
This was causing problems for typings with the @types/dockerode update and the issue it works around was fixed in https://github.com/apocas/docker-modem/commit/6c346d131f1d8d172909f5b4d517a6425d8d2282 and is definitely included as we're running 3.x